framework/db/oci/Schema.php

```diff
@@ -219,7 +224,21 @@ protected function createColumn($column)
             if (stripos($column['DATA_DEFAULT'], 'timestamp') !== false) {
                 $c->defaultValue = null;
             } else {
-                $c->defaultValue = $c->phpTypecast($column['DATA_DEFAULT']);
+                $defaultValue = $column['DATA_DEFAULT'];
+                if ($c->type === 'timestamp' && $defaultValue === 'CURRENT_TIMESTAMP') {
+                    $c->defaultValue = new Expression('CURRENT_TIMESTAMP');
+                } else {
+                    if ($defaultValue !== null) {
+                        if (($len = strlen($defaultValue)) > 2 && $defaultValue[0] === "'"
+                            && $defaultValue[$len - 1] === "'"
+                        ) {
+                            $defaultValue = substr($column['DATA_DEFAULT'], 1, -1);
```

Comment:
We've already discussed this. Also, other schemas just trim quotes here which is invalid and could remove too much. Should I report it in another issue?